LPFilter/Source/PluginProcessor.h

```diff
@@ -61,13 +61,20 @@ class LpfilterAudioProcessor  : public AudioProcessor
 
 private:
     
-    void process (dsp::ProcessContextReplacing<float>) noexcept;
+    void process (AudioSampleBuffer& buffer) noexcept;
```

Comment:
It would be better to create 3 different process methods for each case.
So, create 
- juceModulesProcess(dsp::ProcessContextReplacing<float>)
- dspFiltersProcess(AudioSampleBuffer& buffer)
- customProcess(...)